### PR TITLE
ParamControl dialog: more explicit confirmation text

### PR DESCRIPTION
**Pull request by sshane**

### CODE REVIEW

File: selfdrive/ui/translations/main_ko.ts [hunk 1]
```diff
@@ -473,14 +473,14 @@ location set</source>
...
-        <translation>확인</translation>
-    </message>
     <message>
         <source>Cancel</source>
         <translation>취소</translation>
     </message>
+    <message>
+        <source>Enable</source>
+        <translation>사용</translation>
+    </message>
```
Comment (sshane) [inline selfdrive/ui/translations/main_ko.ts:483]: @crwusiz is this correct?

File: selfdrive/ui/translations/main_pt-BR.ts [hunk 2]
```diff
@@ -474,14 +474,14 @@ trabalho definido</translation>
...
-        <translation>OK</translation>
-    </message>
     <message>
         <source>Cancel</source>
         <translation>Cancelar</translation>
     </message>
+    <message>
+        <source>Enable</source>
+        <translation>Ativar</translation>
+    </message>
```
Comment (sshane) [inline selfdrive/ui/translations/main_pt-BR.ts:484]: @AlexandreSato does this look right?

File: selfdrive/ui/translations/main_pt-BR.ts [hunk 3]
```diff
@@ -377,7 +377,7 @@
     <message>
         <source>Get turn-by-turn directions displayed and more with a comma
 prime subscription. Sign up now: https://connect.comma.ai</source>
-        <translation>Obtenha instruções passo a passo exibidas e muito mais com 
+        <translation>Obtenha instruções passo a passo exibidas e muito mais com
```
Comment (sshane) [inline selfdrive/ui/translations/main_pt-BR.ts:380]:
```suggestion
        <translation>Obtenha instruções passo a passo exibidas e muito mais com 
```

File: selfdrive/ui/translations/main_pt-BR.ts (hunk 2 above)
Comment (AlexandreSato) [reply selfdrive/ui/translations/main_pt-BR.ts:484]: "ativar" or "habilitar" looks very good!

File: selfdrive/ui/translations/main_ko.ts (hunk 1 above)
Comment (crwusiz) [reply selfdrive/ui/translations/main_ko.ts:483]: yes correct.~

File: selfdrive/ui/translations/main_pt-BR.ts (hunk 3 above)
Comment (sshane) [inline selfdrive/ui/translations/main_pt-BR.ts:380]:
```suggestion
        <translation>Obtenha instruções passo a passo exibidas e muito mais com 
```

**Pull request merged**